src/Sentry.AspNetCore/SentryMiddleware.cs

```diff
@@ -47,7 +49,8 @@ internal class SentryMiddleware
             Func<IHub> hubAccessor,
             IOptions<SentryAspNetCoreOptions> options,
             IHostingEnvironment hostingEnvironment,
-            ILogger<SentryMiddleware> logger)
+            ILogger<SentryMiddleware> logger,
+            ISentryClient sentryClient)
```

Comment:
Don't take `ISentryClient` here. Please use `hubAccessor()` instead:

`IHub` extends `ISentryClient` while adding scope management capabilities.
This class uses sentry via `IHub` already (while getting the instance via `hubAccessor` on each use).
Because of that there's no reason to get `ISentryClient` injected.